float32 Voltage
float32 Current